assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
+// --------------------------------------------------
+
+label.checkbox {
+  font-weight: normal;
+}
+
+.form-group .checkbox {
+  // Use the default form-group margin for consistency. Otherwise the
+  // checkboxes are closer than other form elements.
+  margin-bottom: inherit;
+}
+
+// Make sure icons are the same size so headings align.
```

Comment:
This comment might have been here before, but I'm not sure what it's referring to.